src/modules/auth/components/SignUpForm.tsx

```diff
@@ -55,14 +65,13 @@ export function SignUpForm() {
           rules={{ required: true }}
         />
 
-        <InputField<SignUpSchemaType>
+        <PasswordInput<SignUpSchemaType>
           className={"col-span-12 lg:col-span-6"}
           control={control}
-          label={"E-mail"}
-          name={"email"}
-          placeholder={"Type your e-mail"}
+          label={"Password Confirmation"}
```

Comment:
En los otros inputs los tenemos así, debería ser un estandar del front, o cada palabra arranca en mayuscula o solo la primer palabra y el resto minuscula

```suggestion
          label={"Password confirmation"}
```